app/controllers/application_controller.rb

```diff
@@ -1,2 +1,8 @@
 class ApplicationController < ActionController::API
+  include DeviseTokenAuth::Concerns::SetUserByToken
+
+  def error_message(johan)
+    error_message = johan.full_messages.to_sentence
```

Comment:
johan?

Reply:
I thought I changed this before, jenny spotted it as well but somehow we missed to change it, will change it now!